src/boot/efi/stub.c

```diff
@@ -103,6 +103,13 @@ static EFI_STATUS combine_initrd(
 }
 
 static void export_variables(EFI_LOADED_IMAGE_PROTOCOL *loaded_image) {
+        static const uint64_t stub_features =
+                EFI_STUB_FEATURE_REPORT_BOOT_PARTITION |    /* We set LoaderDevicePartUUID */
+                EFI_STUB_FEATURE_PICK_UP_CREDENTIALS |      /* We pick up credentials from the boot partition */
+                EFI_STUB_FEATURE_PICK_UP_SYSEXTS |          /* We pick up system extensions from the boot partition */
```

Comment:
If you add feature flags for these, the dtb feature is recent too, no?

Reply:
the feature flags are supposed to indicate stuff to userspace that it can make programmatical use of. i.e. it's more about reporting OS facing *API* features of the stub than just features of the stub that face elsewhere. I am not sure how an app running on the OS would take benefit of the dtb loading stuff, given the only way to get the dtb loaded is by linking it in via objcopy... it's a kernel builder facing feature, sure. But userspace API? i don't see it.